packages/build/tests/core/fixtures/esbuild_errors/netlify.toml

```diff
@@ -1,5 +1,3 @@
-[build]
-functions = "functions"
-
```

Comment:
We're still keeping support for this specific configuration correct? (i.e.`build.functions`) If so, should we (or maybe we're already keeping) a fixture that exercises that option?

Reply:
> We're still keeping support for this specific configuration correct?

That is correct. I was relying on the tests on the `@netlify/config` side, which can be seen in https://github.com/netlify/build/pull/2496, to assert that `build.functions` is still translated to `functionsDirectory`. I'm not keeping any fixtures for that case on the `@netlify/build` side, because I'm not sure which ones we'd do (i.e. we don't want to duplicate every fixture to account for the two properties).

What do you think?

Reply:
Ah good point! 👍 I started reviewing this PR first 😄 Makes sense to me, as long as we've got some fixture/test covering it I'm fine with it.